lisa/tools/stress_ng.py

```diff
@@ -47,6 +47,18 @@ def launch(
             cmd += f" --timeout {timeout_in_seconds} "
         self.run(cmd, force_run=True)
 
+    def launch_cpu(
+        self, num_cores: int = 0, timeout_in_seconds: int = 0
+    ) -> None:
+        # --cpu N, start N CPU workers
+        # --timeout T, timeout after T seconds
+        cmd = ""
+        if num_cores:
+            cmd += f" --cpu {num_cores} "
+        if timeout_in_seconds:
+            cmd += f" --timeout {timeout_in_seconds} "
+            self.run(cmd, force_run=True, timeout=timeout_in_seconds)
```

Comment:
```       
        if num_cores:
            cmd += f" --cpu {num_cores} "
        if timeout_in_seconds:
            cmd += f" --timeout {timeout_in_seconds} "
        self.run(cmd, force_run=True, timeout=timeout_in_seconds)
```

Reply:
There are hanging spaces at the end of the strings